File: java/Processor.java
public class Processor extends Komponen {
    private String namaprocessor;
    private String generasi;
    private double kecepatanGhz;

    public Processor() {
        super();
        this.namaprocessor = "";
        this.generasi = "";
        this.kecepatanGhz = 0.0;
    }

    public Processor(String merk, String nama, String namaprocessor, String generasi, double kecepatanGhz) {
        super(merk, nama);
        this.namaprocessor = namaprocessor;
        this.generasi = generasi;
        this.kecepatanGhz = kecepatanGhz;
    }

    // Getter
    public String getNamaprocessor() {
        return namaprocessor;
    }

    public String getGenerasi() {
        return generasi;
    }

    public double getKecepatanGhz() {
        return kecepatanGhz;
    }

    // Setter
    public void setNamaprocessor(String namaprocessor) {
        this.namaprocessor = namaprocessor;
    }

    public void setGenerasi(String generasi) {
        this.generasi = generasi;
    }

    public void setKecepatanGhz(double kecepatanGhz) {
        this.kecepatanGhz = kecepatanGhz;
    }

    // Method info
    public String info() {
        return "Processor: " + this.namaprocessor + " " + this.generasi + " " + this.kecepatanGhz + " GHz (Merk: " + this.getMerk() + ")";
    }
}

File: Python/Processor.py
from Komponen import Komponen

class Processor(Komponen):
    def __init__(self, merk="", nama="", namaprocessor="", generasi="", kecepatanGhz=0.0):
        super().__init__(merk, nama)
        self.namaprocessor = namaprocessor
        self.generasi = generasi
        self.kecepatanGhz = kecepatanGhz

    # Getter
    def get_namaprocessor(self):
        return self.namaprocessor

    def get_generasi(self):
        return self.generasi

    def get_kecepatanGhz(self):
        return self.kecepatanGhz

    # Setter
    def set_namaprocessor(self, namaprocessor):
        self.namaprocessor = namaprocessor

    def set_generasi(self, generasi):
        self.generasi = generasi

    def set_kecepatanGhz(self, kecepatanGhz):
        self.kecepatanGhz = kecepatanGhz

    # Method info
    def info(self):
        return f"Processor: {self.namaprocessor} {self.generasi} {self.kecepatanGhz} GHz (Merk: {self.get_merk()})"